### DISCUSSION TITLE
vbscript support / converter

### DISCUSSION THREAD

**Discussion by kirillFedorov1** [ONLYOFFICE/DocumentServer#1672] [Ideas]
I am currently working for a company which is going to switch from ms office to onlyoffice. They have 800 documents, some of it are with ms office macroses, and it means there are probably tens of thousands of lines of vbscript code. That exact company perhaps doesn't mind to pay me for manually converting all the scripts, but if there is a company with, let's say, 8000 documents - it is coming to my mind that vbscript support / converter could be critical or atleast desirable.

**Comment by ShockwaveNN** [comment]
Hi

We fought about writing some kind of converter but seems that syntax of methods are completely different and this product will be 
a. very complicated
b. will not very well

We still can't say that this converter never will be implemented, but right now we have no prototype or any kind of proof of concept

**Reply by kirillFedorov1** [reply]
Sad, if you tried to come up with it, are there any kind of tips or links about how to automatize the routine atleast partly?

**Reply by ShockwaveNN** [reply]
Sorry, I don't think so
